scripts/deploy-federation.sh

```diff
@@ -99,10 +99,16 @@ IMAGE_NAME="${1:-}"
 NAMESPACED="${NAMESPACED:-}"
 
 LATEST_IMAGE_NAME=quay.io/kubernetes-multicluster/federation-v2:latest
+CANARY_IMAGE_NAME=quay.io/kubernetes-multicluster/federation-v2:canary
```

Comment:
In https://github.com/kubernetes-sigs/federation-v2/pull/716/commits/b270b3c5c3ae3f74d8737db3a7e2be22fa4297a6, `quay.io/kubernetes-multicluster/federation-v2:canary` is changed to `<containerregistry>/<username>/federation-v2:canary`. I think it makes sense to build your own image to test latest master changes instead of using existing one. 

So, I guess it is better to keep old logic without `canary` part here. @shashidharatd

Reply:
Okay, I've reverted the `canary` part.